app/controllers/v0/intent_to_files_controller.rb

```diff
@@ -64,6 +64,16 @@ def submit
 
     private
 
+    def authorize_service
+      # Is this necessary if we've fully migrated to Lighthouse? EVSS tests still exist in the request spec,
+      # so it might be necessary until those are removed
+      if Flipper.enabled?(ApiProviderFactory::FEATURE_TOGGLE_INTENT_TO_FILE, @current_user)
```

Comment:
I am checking if this flipper is still required; we can remove it in a flipper cleanup ticket later if no longer needed